### PR TITLE
minimap: different color and outline for player arrow

### PR DESCRIPTION
**Pull request by sweet235**
Since we added `cg_minimapRotate`, the player arrow on the minimap is not always pointing up anymore. Increase its visibility by making it orange with a black outline. Screenshots of map-terminus for comparison:

Before:
unvanquished_2024-07-04_133840_000

After:
unvanquished_2024-07-04_133944_000

### CODE REVIEW

**Review by VReaperV — APPROVED** [review]
Having played the game with this change on map&bot testing server for a while, this is much better than what we had before. Code also LGTM.

**Review by cu-kai — APPROVED** [review]
LGTM; good improvement.

File: src/cgame/cg_minimap.cpp [hunk 1]
```diff
@@ -437,7 +437,14 @@ CG_MinimapDrawPlayer
 */
 static void CG_MinimapDrawPlayer( const minimap_t* m )
 {
+	Color::Color oldColor = currentMinimapColor;
+	// draw outline
+	currentMinimapColor = { 0.f, 0.f, 0.f, 1.f };
+    CG_DrawMinimapObject( m->gfx.playerArrow, cg.refdef.vieworg, cg.refdefViewAngles[1], 1.2, MINIMAP_PLAYER_DISPLAY_SIZE, 1.0 );
```
Comment (DolceTriade) [inline src/cgame/cg_minimap.cpp:443]: indentation is wonky here and below.

**Comment by Grise3** [comment]
Great! The minimap is really great with that!

**Comment by DolceTriade** [comment]
lgtm

**Review by illwieckz — APPROVED** [review]
LGTM. Don't forget to squash. 😉️

**Pull request merged**